src/app/core/models/label.model.ts

```diff
@@ -1,4 +1,74 @@
 export interface Label {
-    labelValue: string;
-    labelColor: string;
-  }
+  labelValue: string;
+  labelColor: string;
+  labelCategory: LABEL_CATEGORY;
+}
+
+export enum LABEL_CATEGORY {
+  severity = 'severity',
+  type = 'type',
+  response = 'response',
+  status = 'status'
+}
+
+export enum CATEGORY_STATUS_VALUES {
```

Comment:
A question,

The default label names and categories seem to be currently stored in the [`issue.model.ts`](https://github.com/ptvrajsk/CATcher/blob/LabelSetup/src/app/core/models/issue.model.ts) (Which I have replicated here). Should I shift the label specific information to the [`label.model.ts`](https://github.com/ptvrajsk/CATcher/blob/LabelSetup/src/app/core/models/label.model.ts) from [`issue.model.ts`](https://github.com/ptvrajsk/CATcher/blob/LabelSetup/src/app/core/models/issue.model.ts) ? Perhaps in another PR?

Reply:
@JunWei96 what do you think?

Reply:
Yeap, I think it would be more suitable to be in `label.model.ts`.